Batch File/Batch File.sublime-syntax

```diff
@@ -20,12 +21,18 @@ contexts:
       scope: keyword.control.conditional.dosbatch
     - match: \b(?i)for\b
       scope: keyword.control.repeat.dosbatch
-    - match: \b(?:EQU|NEQ|LSS|LEQ|GTR|GEQ)\b
-      scope: keyword.operator.dosbatch
-    - match: "[|]"
+
+  operators:
+    - match: '@(?=\S)'
+      scope: keyword.operator.at.dosbatch
+    - match: \b(?i:EQU|NEQ|LSS|LEQ|GTR|GEQ|==)\b
+      scope: keyword.operator.comparison.dosbatch
+    - match: '&&?|\|\|'
```

Comment:
`&` is for running a second command regardless of the outcome of the first. It's equivalent to `;` in bash which is scoped as `keyword.operator.list.shell` in `Shell-Unix-Generic`. Everything I've read<sup><a href="http://www.robvanderwoude.com/condexec.php">1</a></sup> <sup><a href="http://ss64.com/nt/syntax-conditional.html">2</a></sup> <sup><a href="https://www.microsoft.com/resources/documentation/windows/xp/all/proddocs/en-us/ntcmds_shelloverview.mspx?mfr=true">3</a></sup> has associated `&` with the conditional operators.